aurora/core.py

```diff
@@ -121,26 +131,57 @@ def reload_namelist(self, namelist=None):
         """
         if namelist is not None:
             self.namelist = namelist
+            
+        # Extract one by one all the inputs from namelist
+        # as attributes of asim, keeping the same name  
+        for parameter in self.namelist:
+            if isinstance(self.namelist[parameter], dict):
+                for sub_parameter in self.namelist[parameter]:
+                     setattr(self, sub_parameter, self.namelist[parameter][sub_parameter])
```

Comment:
@odstrcilt I have mixed feelings about this too, but I advised @antonellozito to try this because otherwise there was a long list of things like `self.mykey = self.namelist["mykey"]`, which didn't add much. 

The fact that parameters in the namelist may have the same name as an Aurora parameter should be prevented. This could be done simply by adding an if-statement of the sort `if hasattr(self, "parameter"): raise ValueError("Input namelist parameter is trying to overwrite an existing attribute!")`.

Reply:
@odstrcilt exactly, I did this just in order to condense lines. The other possibility would be just not dumping the namelist parameters as attributes, and call them explicitly, i.e. using later in core.py s`elf.namelist["mykey"]` instead of `self.mykey`. The reason why I tried to do it is, indeed, to make the lines calling namelist parameters more slim and more easily readable.

@fsciortino of course I agree, but from the testing I have done, it does never happen that namelist parameters have the same name of other variables used in core.py, so no conflicts.